src/pages/en/guides/deploy/kinsta.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Deploy your Astro Site to Kinsta Application Hosting
+description: How to deploy your Astro site to the web on Kinsta Application Hosting.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can use [Kinsta Application Hosting](https://kinsta.com/application-hosting/) to host an Astro site on their cloud hosting.
+
+:::tip[Looking for an example?]
+Check out [the official Kinsta Application Hosting Starter project for Astro](https://github.com/kinsta/hello-world-astro)!
+:::
+
+## Project configuration
+To host your project on **Kinsta Application Hosting**, you need to:
+- make sure your `package.json` has a `name` field.
+- have a script called `build` in your `package.json.`
+- install the `serve` package and set the `start` script to `serve dist/`.
+
+This is an example of a correct `package.json` file:
+```json
+{
+  "name": "@example/blog",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "serve dist/",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "dependencies": {
+    "astro": "^1.1.7",
+    "@astrojs/mdx": "^0.11.1",
+    "@astrojs/rss": "^1.0.0",
+    "@astrojs/sitemap": "^1.0.0",
+    "serve": "^14.0.1"
+  },
+  "devDependencies": {
+    "astro": "^1.1.7",
+    "@astrojs/renderer-preact": "^0.5.0"
+  }
+}
+```
```

Comment:
``````suggestion
```json title="astro.config.mjs" {2,6} ins={12} "serv dist/"
{
  "name": "@example/blog",
  "scripts": {
    "dev": "astro dev",
    "start": "serve dist/",
    "build": "astro build",
    "preview": "astro preview",
    "astro": "astro"
  },
  "dependencies": {
    "astro": "^1.6.10",
    "serve": "^14.0.1"
  },
}
```
``````
Unless those other dependencies are required, I would not show them here. The version of Astro (and others) was also very old, so it is better to show as little as possible. I also highlighted the contents.